cirrus/google_cloud/manager.py

```diff
@@ -661,6 +662,38 @@ def give_group_access_to_bucket(self, group_email, bucket_name, access=None):
 
         bucket.update()
 
+    @backoff.on_exception(backoff.expo, Exception, **BACKOFF_SETTINGS)
+    def delete_data_file(self, bucket_name, object_name):
+        """
+        Delete a file within the provided bucket with the provided file ID.
+
+        Args:
+            bucket_name (str): name of Google Cloud Storage bucket containing file to delete
+            file_name (str): name of file to delete
+
+        Returns:
+            A tuple, (dict, status)
+
+            dict: is JSON response from API call, which should be empty if
+                    it successfully deleted the file
+            `Google API Reference <https://cloud.google.com/storage/docs/deleting-objects#rest-delete-object>`_
+
+            status: status code of response
+        """
+        api_url = _get_google_api_url(
+            "b/" + bucket_name + "/o/" + object_name, GOOGLE_STORAGE_API_URL
```

Comment:
if this is what needs to be percent-encoded, you should do it here instead of in fence so when people use it in the future they don't have to know about that